playbooks/bootstrap-python.yml

```diff
@@ -4,7 +4,14 @@
   gather_facts: False
   tasks:
     # Fortunately, python is a dependency of yum. So RedHat systems always have it.
-    - name: install python
+    - name: install python (no proxy)
       raw: test -e /usr/bin/python || (apt -y update && apt install -y python-minimal)
       register: output
       changed_when: output.stdout != ""
+      when: proxy_env is not defined
+
+    - name: install python (using proxy)
+      raw: test -e /usr/bin/python || (https_proxy="{{ https_proxy }}" http_proxy="{{ http_proxy }}" apt -y update && https_proxy="{{ https_proxy }}" http_proxy="{{ http_proxy }}" apt install -y python-minimal)
```

Comment:
Can we use `proxy_env['http_proxy']` and `proxy_env['https_proxy']` here instead of the named variables on their own? This will avoid the duplication in the group_vars file.

Reply:
unfortunately, the ansible command does not respect the group_vars and this was the only workaround I was able to find.